recipes/jam/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "jam" %}
+{% set version = "2.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://swarm.workshop.perforce.com/downloads/guest/perforce_software/jam/jam-{{ version }}.zip
+  sha256: 7c510be24dc9d0912886c4364dc17a013e042408386f6b937e30bd9928d5223c
+  # Removed CC and CFLAG in Makefile 
+  patches: remove_cc.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+  script:
+    - export AR="ar r"  # [unix]
```

This is using `ar` from the host. I think this works:
```suggestion
    - export AR="${AR} r"  # [unix]
```

---

Indeed! My mistake, thanks!